contrib/metrics/supply_check/supply_deltas.py

```diff
@@ -0,0 +1,60 @@
+import pprint
+import bisect
+
+class SupplyDeltas:
+    def __init__(self, zcashd, fr_addrs, miner_deltas, flush_interval = 500):
+        self.zcashd = zcashd
+        self.fr_addrs = fr_addrs
+        self.miner_deltas = miner_deltas
+
+        self.delta_count = 0
```

Comment:
`self.delta_count` is always equal to `len(self.delta_cache)` (except temporarily in `AddSupplyDelta`), so the instance variable can be eliminated.

Reply:
Ah, yeah; I had originally used a dict for the cache, but then I realized that it needed to be an ordered sequence of pairs instead.